### PR TITLE
fix(p-json-schema-form): add slots for dropdown components

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console(if usages are changed) 

### Description
스크린샷 2022-12-13 오후 2 06 04

### CODE REVIEW

File: src/inputs/forms/json-schema-form/story-helper.ts [hunk 1]
```diff
@@ -97,6 +97,40 @@ export const getJsonSchemaFormArgTypes = (): ArgTypes => ({
...
+                summary: null,
+            },
+            defaultValue: {
+                summary: null,
+            },
+            category: 'slots',
+        },
+        control: {
+            type: 'text',
+        },
```
Comment (WANZARGEN) [inline src/inputs/forms/json-schema-form/story-helper.ts:115]: suggestion: what about adding just one slot, 'dropdown-extra' ?

File: src/inputs/forms/json-schema-form/mock.ts [hunk 2]
```diff
@@ -78,13 +78,25 @@ export const getDefaultSchema = () => ({
...
+        color: {
+            description: 'Favorite Color',
+            title: 'Favorite Color (single input)',
+            type: 'array',
+            prefixItems: [
+                { type: 'string' },
+                { enum: ['red', 'blue', 'gold', 'purple', 'gray'] },
+            ],
+            default: ['gray'],
+            multiInputMode: false,
```
Comment (WANZARGEN) [inline src/inputs/forms/json-schema-form/mock.ts]: praise: haha 😂 I know very well that you must have been thinking about what kind of property to put in. Favorite Color is very goood 😄

File: src/inputs/forms/json-schema-form/mock.ts [hunk 3]
```diff
@@ -78,13 +78,25 @@ export const getDefaultSchema = () => ({
...
+            multiInputMode: true,
+        },
+        color: {
+            description: 'Favorite Color',
+            title: 'Favorite Color (single input)',
+            type: 'array',
+            prefixItems: [
+                { type: 'string' },
+                { enum: ['red', 'blue', 'gold', 'purple', 'gray'] },
+            ],
```
Comment (WANZARGEN) [inline src/inputs/forms/json-schema-form/mock.ts]: suggestion: I strongly recommend to add description about `prefixItems` and `multiInputMode` keyword into storybook.

image

**Review by WANZARGEN — COMMENTED** [review]
Nice clean code :)
I strongly recommend to update storybook docs :)

File: src/inputs/forms/json-schema-form/story-helper.ts (hunk 1 above)
Comment (yuda110) [reply src/inputs/forms/json-schema-form/story-helper.ts:115]: good idea

File: src/inputs/forms/json-schema-form/helper.ts [hunk 4]
```diff
@@ -153,3 +153,8 @@ export const getMenuItemsBySchemaProperty = (schemaProperty: InnerJsonSchema): S
     }
     return undefined;
 };
+
+export const getMultiInputMode = (schemaProperty: InnerJsonSchema): boolean => {
+    if (schemaProperty.type !== 'array') return false;
+    return schemaProperty?.maxItems !== 1;
+};
```
Comment (WANZARGEN) [inline src/inputs/forms/json-schema-form/helper.ts:160]: praise: how nice :)

**Review by WANZARGEN — APPROVED** [review]
What a beautiful documentation!! I TRUELY love it 💓 
LGTM!

**Pull request merged**